File: vote-service/src/main/java/br/com/sicredi/api/exception/error/ErrorResponse.java
package br.com.sicredi.api.exception.error;

import com.fasterxml.jackson.annotation.JsonFormat;

import java.time.Instant;

public record ErrorResponse(
        @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd/MM/yyyy - HH:mm:ss", timezone = "America/Sao_Paulo")
        Instant timestamp,
        Integer statusCode,
        String error,
        String message,
        String path
) {
}

File: validate-random-cpf/src/main/kotlin/br/com/sicredi/generate_cpf/exception/ErrorResponse.kt
package br.com.sicredi.generate_cpf.exception

import com.fasterxml.jackson.annotation.JsonFormat
import java.time.Instant

data class ErrorResponse(
    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd/MM/yyyy - HH:mm:ss", timezone = "America/Sao_Paulo")
    val timestamp: Instant,
    val statusCode: Int,
    val error: String,
    val message: String,
    val path: String
)
